src/utilities/valueFactory.js

```diff
@@ -16,17 +17,31 @@ const newValue = (
   uri,
   label,
   valueSubject,
+  component,
 })
 
 export const newLiteralValue = (property, literal, lang = defaultLanguageId) =>
-  newValue(property, literal, lang, null, null, null)
+  newValue(property, literal, lang, null, null, null, "InputLiteralValue")
+
+export const newBlankLiteralValue = (property) =>
+  newValue(
+    property,
+    null,
+    defaultLanguageId,
+    null,
+    null,
+    null,
+    "InputLiteralValue"
```

Comment:
that's a lot of args to remember in an ordered way.  are keyword args possible in javascript?

Reply:
I agree this isn't awesome. However, since newValue is only used within `valueFactory` and the methods that are exported have fewer args, this seems less horrible.